eclass/virtualx.eclass

```diff
@@ -33,17 +32,16 @@ _VIRTUAL_X=1
 # @DESCRIPTION:
 # Dep string available for use outside of eclass, in case a more
 # complicated dep is needed.
-# You can specify the variable BEFORE inherit to add more dependencies.
-VIRTUALX_DEPEND="${VIRTUALX_DEPEND}
+# DEPRECATED: Pre-EAPI-8 you can specify the variable BEFORE inherit
+# to add more dependencies.
+[[ ${EAPI} == [67] ]] && VIRTUALX_DEPEND="${VIRTUALX_DEPEND}"
```

Comment:
Err, you do realize that this doesn't do anything, right? ;-)

Reply:
rright...